bn-apps/cordapp-updates-distribution/corda-updates-transport/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'kotlin'
```

Comment:
Can you maybe break the flows transport out into a separate PR, because this one is exhausting to review. Please don't make such huge PRs again!

Reply:
Apologies for that. I moved corda-updates-shell and corda-updates-transport to a separate branch. Will submit them later on in separate PRs. corda-based transports are mentioned in the README file and in some KDocs. I added statement about that  to the beginning of the README.